### PR TITLE
Feature/loader component

### PR DESCRIPTION
**Pull request by humayunadilshahzad**

### CODE REVIEW

**Comment by senner008** [comment]
@humayunadilshahzad 
If no new packages are introduced then package-lock should not be committed either

**Comment by humayunadilshahzad** [comment]
Hi Niels .. actually my storybook was not running I did new pull and did npm install ... which modified package-lock.json .. thats y i commited it ..
Is it not needed to commit ?

**Comment by senner008** [comment]
No you should not commit it. You need run your install using the one from develop

**Comment by senner008** [comment]
You can try to undo your commits or create a new branch

**Comment by humayunadilshahzad** [comment]
thats what I was thing ... can I remove a specific commit or it has to be in reverse order ..
I guess I can undo any commit .. yes ?

**Comment by senner008** [comment]
https://stackoverflow.com/questions/22682870/git-undo-pushed-commits

**Comment by senner008** [comment]
Alternatively just overwrite your package-lock with the one from develop and push that commit

**Comment by humayunadilshahzad** [comment]
Niels I have reverted it locally . I did 
 "  git revert 0184a40c7fabd951c94c5f789302f032d0f0f09d "
it says  [feature/loader-component 96d2f6d] Revert "packages updated"
 1 file changed, 69 insertions(+), 23 deletions(-) .. 
documentation says .. revert commit ... is a new commit ..
do I need to push it again ?

**Comment by senner008** [comment]
Yes, push the commit

**Comment by senner008** [comment]
That did the trick 👍

**Comment by humayunadilshahzad** [comment]
Thanx Niels

File: src/client/components/loader-animation/loader-animation.style.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.loader {
+  position: absolute;
+  top: 47%;
+  left: 47%;
+}
+
+.bubble {
+  animation: expand 1s ease-in-out infinite;
```
Comment (cecastosic) [inline src/client/components/loader-animation/loader-animation.style.css:12]: Can we try with a little bit faster animation? 
```suggestion
  animation: expand .8s ease-in-out infinite;
```

File: src/client/components/loader-animation/loader-animation.style.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  border-radius: 10px;
+  display: inline-block;
+  transform-origin: center center;
+  margin: 0 3px;
```
Comment (cecastosic) [inline src/client/components/loader-animation/loader-animation.style.css:16]: and a little bit bigger margin
```suggestion
  margin: 0 5px;
```

**Comment by cecastosic** [comment]
Great job @humayunadilshahzad 💪

**Comment by senner007** [comment]
@cecastosic @humayunadilshahzad 
Is it ready for merge?

**Comment by humayunadilshahzad** [comment]
Thanx ..

File: src/client/components/loader-animation/loader-animation.style.css (hunk 1 above)
Comment (humayunadilshahzad) [reply src/client/components/loader-animation/loader-animation.style.css:12]: yes we can do it .. I just liked it in easy and relaxed way .. hehe

**Pull request merged**